database/seeders/DevelopmentDatabaseSeeder.php

```diff
@@ -79,5 +80,7 @@ public function run(): void
         ]);
 
         Message::factory()->count(10)->recycle($tickets)->create(['author_id' => $user->id]);
+
+        Bank::factory()->count(10)->create();
```

Comment:
There is a BankSeeder no need to mock

Reply:
I was going to use it but it gave me an error and I factory deployed it because `DevelopmentDatabaseSeeder.php` is for development purposes only.

However, I have checked the `BankSeeder` to be able to use it and since the `uuid` is obtained using the `bic`, there are repeated keys: BBPIPTPL and COBADEFFXXX, so it gives an error since the keys must be unique.

Reply:
BankSeeder fixed!